admin/includes/admin_html_head.php

```diff
@@ -14,26 +14,33 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <title><?php echo TITLE; ?></title>
 <?php if (file_exists($file = 'includes/css/bootstrap.min.css')) { ?>
-    <link rel="stylesheet" href="<?php echo $file; ?>">
+  <link rel="stylesheet" href="<?php echo $file; ?>">
 <?php } else { ?>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
 <?php } ?>
 <?php if (file_exists($file = 'includes/css/font-awesome.min.css')) { ?>
-    <link rel="stylesheet" href="<?php echo $file; ?>">
+  <link rel="stylesheet" href="<?php echo $file; ?>">
 <?php } else { ?>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
 <?php } ?>
 <?php if (file_exists($file = 'includes/css/jquery-ui.css')) { ?>
-    <link rel="stylesheet" href="<?php echo $file; ?>">
+  <link rel="stylesheet" href="<?php echo $file; ?>">
 <?php } else { ?>
-    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+  <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
 <?php } ?>
-    <link rel="stylesheet" href="includes/css/jAlert.css">
-    <link rel="stylesheet" href="includes/css/menu.css">
-    <link rel="stylesheet" href="includes/css/stylesheet.css">
+<link rel="stylesheet" href="includes/css/jAlert.css">
+<link rel="stylesheet" href="includes/css/menu.css">
+<link rel="stylesheet" href="includes/css/stylesheet.css">
 <?php if (file_exists($file = 'includes/css/' . basename($PHP_SELF, '.php') . '.css')) { ?>
-    <link rel="stylesheet" href="<?php echo $file; ?>">
-<?php } ?>
+  <link rel="stylesheet" href="<?php echo $file; ?>">
 <?php
+}
+
+$template = new template_func();
```

Comment:
Isn't `$template` already instantiated by the config loader?

Reply:
Didn't know that.
 lines are removed